torch/onnx/symbolic_helper.py

```diff
@@ -408,18 +408,35 @@ def _interpolate_warning(interpolate_mode):
                   "We recommend using opset 11 and above for models using this operator.")
 
 def _unsqueeze_helper(g, input, axes_i):
-    if _export_onnx_opset_version >= 13:
-        axes = g.op("Constant", value_t=torch.tensor(axes_i, dtype=torch.long))
-        return g.op("Unsqueeze", input, axes)
-    else:
-        return g.op("Unsqueeze", input, axes_i=axes_i)
+    if _is_constant(axes_i[0]):
+        if _export_onnx_opset_version >= 13:
+            axes = g.op("Constant", value_t=torch.tensor(axes_i, dtype=torch.long))
+            return g.op("Unsqueeze", input, axes)
+        else:
+            return g.op("Unsqueeze", input, axes_i=axes_i)
+    else:  # Tensor type
+        if _export_onnx_opset_version >= 13:
+            axes_t = axes_i[0]
+            return g.op("Unsqueeze", input, axes_t)
+        else:
+            raise ValueError("Unsupported Unsqueeze dynamic axes ONNX opset version: " + str(_export_onnx_opset_version))
 
 def _squeeze_helper(g, input, axes_i):
-    if _export_onnx_opset_version >= 13:
-        axes = g.op("Constant", value_t=torch.tensor(axes_i, dtype=torch.long))
-        return g.op("Squeeze", input, axes)
-    else:
-        return g.op("Squeeze", input, axes_i=axes_i)
+    if _is_constant(axes_i[0]):
+        if _export_onnx_opset_version >= 13:
+            axes = g.op("Constant", value_t=torch.tensor(axes_i, dtype=torch.long))
+            return g.op("Squeeze", input, axes)
+        else:
+            return g.op("Squeeze", input, axes_i=axes_i)
+    else:  # Tensor type
+        if _export_onnx_opset_version >= 13:
+            axes_t = axes_i[0]
+            if _get_tensor_sizes(axes_t) and _get_tensor_sizes(axes_t) == 1:
+                return g.op("Squeeze", input, axes_t)
+            else:
+                raise ValueError("For squeeze axses as input, the dimension must be one in ONNX spec")
```

Comment:
In ONNX spec [squeeze-13](https://github.com/onnx/onnx/blob/main/docs/Changelog.md#squeeze-13), `squeeze` axses dimension must be one. 
So it doesn't easy to add the tests. Cause If we add dim as input, just like unsqueeze test in this PR. The tensor block input `prim::Param` dimension will be `0` and throw the error.

Reply:
can we check if it is scalar, and if so apply unsqueeze to make it rank 1?